windows-driver-docs-pr/devtest/28726-banned-api-usage-use-updated-function-replacement.md

```diff
@@ -9,11 +9,56 @@ f1_keywords:
 # C28726
 
 
-warning C28726: Banned API Usage
+**warning C28726: Banned API Usage L2**\
+_Banned API Usage:  *function name* is a Banned API as listed in dontuse.h for security purposes._
```

Comment:
Is `dontuse.h` a public header? If not, we should remove this reference.

Reply:
I had a conversation offline about this.  The file dontuse.h is public (it is in the windows SDK) so it is okay being referenced.  However, I don't think it is helpful to reference the file.  I am creating a PR on the tool to update the message, after that goes through the message here can be updated to match.